app/src/main/java/chat/rocket/android/chatroom/adapter/ActionsAttachmentViewHolder.kt

```diff
@@ -0,0 +1,44 @@
+package chat.rocket.android.chatroom.adapter
+
+import android.view.View
+import chat.rocket.android.chatroom.uimodel.ActionsAttachmentUiModel
+import chat.rocket.android.emoji.EmojiReactionListener
+import chat.rocket.core.model.attachment.actions.Action
+import chat.rocket.core.model.attachment.actions.ButtonAction
+import kotlinx.android.synthetic.main.item_actions_attachment.view.*
+import androidx.recyclerview.widget.LinearLayoutManager
+import timber.log.Timber
+
+class ActionsAttachmentViewHolder(
+        itemView: View,
+        listener: ActionsListener,
+        reactionListener: EmojiReactionListener? = null,
+        var actionAttachmentOnClickListener: ActionAttachmentOnClickListener
+) : BaseViewHolder<ActionsAttachmentUiModel>(itemView, listener, reactionListener) {
+
+    init {
+        with(itemView) {
+            setupActionMenu(actions_attachment_container)
+        }
+    }
+
+    override fun bindViews(data: ActionsAttachmentUiModel) {
+        val actions = data.actions
+        val alignment = data.buttonAlignment
+        Timber.d("no of actions : ${actions.size} : $actions")
+        with(itemView) {
+            title.text = data.title ?: ""
+            actions_list.layoutManager = LinearLayoutManager(itemView.context,
+                    when (alignment) {
```

Comment:
You can resume the condition in this way:

```
if (alignment == "horizontal") {
   LinearLayoutManager.HORIZONTAL
} else {
   LinearLayoutManager.VERTICAL
}
```
or, if you prefer to use `when`:

```
when (alignment) {
 "horizontal" -> LinearLayoutManager.HORIZONTAL
  else -> LinearLayoutManager.VERTICAL
}
```

Not needed to repeat `LinearLayoutManager.VERTICAL` two times.